backend/app/routers/datasets.py

```diff
@@ -228,7 +228,7 @@ async def get_datasets(
     admin=Depends(get_admin),
     admin_mode: bool = Depends(get_admin_mode),
 ):
-    if admin and admin_mode:
+    if admin and admin_mode and not mine:
```

Comment:
Just double check, based on this logic, we want to completely separate out other people's dataset vs "mine" dataset?

Reply:
Yes. I was thinking that if we enable superadmin, it wouldn't be helpful to have both tabs be the same and just contain all the datasets.